ls1intum/Artemis
# [Feature] Grading System - Student Exam Detail

### Checklist

### Motivation and Context


This PR integrates the grading system on the student exam detail page

### Description

Instructors can now see the student's grade or bonus and whether they have passed in the participant status

### Steps for Testing


1. Log in to Artemis as an instructor and navigate to an exam with existing student submissions.
2. Create a grading key for the exam 
3. Navigate to a student exam detail page for that exam (Student Exams -> View)
4. Check if the grade and whether the student has passed are displayed in the participation status
5. Switch the grading key to 'Bonus' and navigate back to the student exam detail page again
6. Check if the bonus points are displayed
7. Check that the grading information isn't present and that no errors are thrown on the student exam detail page if no grading key exists for the exam

### Test Coverage



student-exam-detail.component.ts: 82.65%

### Screenshots


![Screenshot 2021-07-25 133954](https://user-images.githubusercontent.com/63168603/126897741-bc769a93-0f0e-41c6-8317-0b32aac844c1.png)
![Screenshot 2021-07-25 134034](https://user-images.githubusercontent.com/63168603/126897742-178dbd30-7def-4548-8298-cccc753414d0.png)
